### PR TITLE
updated clustering documentation based on my experience

### PR DESCRIPTION
**Pull request by dneuman64**
I noticed there were a few mistakes in the clustering documentation while I was trying to get clustering to so I fixed them.

Thanks!

### CODE REVIEW

File: content/docs/v0.9/concepts/clustering.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ Configuring a cluster with three host machines A, B, and C:
 2. Generate a config file on each machine by running `influxd config > influxdb.conf`.
 3. Update the `[meta]` section of the configuration file on all three hosts, replacing `localhost` with the hosts actual network IP.
 4. Update the bind-address to another port if 8088 is not acceptable.
-5. On all three machines add `peers = ["IP_address_A:port_A", "IP_address_B:port_B", "IP_address_C:port_C"]` to the `[meta]` section of the config file. For example,
+5. On all three machines add `peers = ["IP_address_A:bind_address_port_A", "IP_address_B:bind_address_port_B", "IP_address_C:bind_address_post_C"]` to the `[meta]` section of the config file. For example,
```
Comment (gunnaraasen) [inline content/docs/v0.9/concepts/clustering.md]: Small typo in `bind_address_post_C`. Should be `bind_address_port_C`

**Comment by gunnaraasen** [comment]
+1 will merge once the typo is fixed.

**Comment by dneuman64** [comment]
good catch, fixed the typo.

**Comment by gunnaraasen** [comment]
Thanks @dneuman64!

**Comment by dneuman64** [comment]
Thanks for merging @gunnaraasen!

**Pull request merged**